### ISSUE TITLE
[APP] Moneybird

### ISSUE THREAD

**Issue by todsac** [PipedreamHQ/pipedream#346]
**Name of app / service**
Moneybird
**Link to developer documentation**
https://developer.moneybird.com/

**Is lack of support preventing you from building workflows, or do you have a workaround?**

**Are there specific actions, or triggers, you'd like to see for this app? Please let us know here or use the Action and Trigger issue templates to open requests for each!**

**Comment by ronald-vonk** [comment]
Thanks! I would like to subscribe to webhooks in Moneybird (new contact, new quotation) but there is no Node.js example.

**Comment by todsac** [comment]
I added the Moneybird app.  Can you confirm it works on your end?

You can add your own actions, such as subscribing to a webhook.  See here
-- https://docs.pipedream.com/workflows/steps/actions/

**Comment by todsac** [comment]
I added the Moneybird app.  Can you confirm it works on your end?

You can add your own actions, such as subscribing to a webhook.  See here -- https://docs.pipedream.com/workflows/steps/actions/

**Comment by ronald-vonk** [comment]
Thanks for your help!
But when I start a new workflow and create a new eventsource I can't select
moneybird, no...

Ronald

Op do 20 aug. 2020 om 18:45 schreef Tod Sacerdoti <notifications@github.com

**Comment by todsac** [comment]
Yes, event sources are a secondary step.  The first integration is as an integrated app. Here is a sample workflow:
https://pipedream.com/@tod/moneybird-integration-example-p_7NCZ9d/edit

**Comment by ronald-vonk** [comment]
Pipedream app connection failure. Please reach out to support@pipedream.com (ERROR ID: 528adb029874426aa399162eb691663b)

**Comment by todsac** [comment]
Ok, i thought that issue might be related to our developer account and not a real account.  I have reached out to Moneybird and will revert back.

**Comment by ronald-vonk** [comment]
Thank you very much. I saw that only invoices are supported. What about contacts?
Best regards,

**Comment by todsac** [comment]
Ok, it appears to be fixed.  You need to enter an administration ID as a parameter - "The :administration_id part indicates the administration you want to access. This id corresponds with the id you see in the path when you are accessing the administration via our web application."

Also, I added all the scopes except time_entries.  For some reason, that one is erroring:  
sales_invoices
documents
estimates
bank
settings

Please try it again.

**Comment by todsac** [comment]
Actually, hold on.  Fixing one more issue.

**Comment by todsac** [comment]
Thanks.  This effort identified another unrelated bug that was important to fix.  It is now out and working end to end for me.  Here is a sample workflow:/
https://pipedream.com/@tod/moneybird-integration-example-2-p_RRCp6j/

**Comment by ronald-vonk** [comment]
?
404: Page not found

**Comment by todsac** [comment]
Sorry, now the page is public.
